Makefile

```diff
@@ -23,12 +23,12 @@ requirements: check_venv
 hello: check_python
 	@echo Looking good!
 
-.PHONY: terraform
-terraform: check_env
-	$(MAKE) -C terraform
-
 .PHONY: deploy
 deploy: check_env
+	$(MAKE) -C terraform
+
+.PHONY: package
```

Comment:
The `package` target should precede the `deploy` target. That matches the natural sequence in which they should be run and would also make this diff hunk more legible.